packages/components/src/utils/validation/partials/validatePartialUsage.ts

```diff
@@ -78,23 +78,30 @@ export const validateGetLoaderScriptUsage = (): void => {
 
 export const validateGetInitialStylesUsage = (): void => {
   if (!document.head.querySelector('style[data-pds-initial-styles]')) {
-    logPartialValidationWarning('getInitialStyles');
+    throwPartialValidationError('getInitialStyles');
   }
 };
 
 export const logPartialValidationWarning = (partialName: PartialName, prefix?: string): void => {
-  consoleWarn(
-    `The Porsche Design System ${
-      prefix ? `with prefix: '${prefix}' ` : ''
-    }is used without using the ${partialName}() partial.`,
-    getWarningRecommendation(partialName)
-  );
+  consoleWarn(getMainText(partialName, prefix), getAdditionalText(partialName));
+};
+
+export const throwPartialValidationError = (partialName: PartialName, prefix?: string): void => {
+  throwException(getMainText(partialName, prefix) + ' ' + getAdditionalText(partialName, true));
+};
+
+export const getMainText = (partialName: string, prefix?: string): string => {
```

Comment:
Could use a more descriptive name especially since its exported e.g. getPartialValidationErrorText

Reply:
Renamed to `getValidatePartialErrorPrimaryText()` and `getValidatePartialErrorSecondaryText()`.